app/Config/config.yml

```diff
@@ -1,5 +1,5 @@
 dev:
```

Comment:
GENERAL COMMENT:
1) Add Validator folder
2) Create Validator interface with single method "validate(Model $model)"
3) create UserValidator and inmplement Validator inteface 
4) move validations to the UserValidator -> validate ($user) method, this method should return array with errors
5) Implement in BaseController validate(Model $model) method
6) BaseController validate(Model $model) should do the following:
- $modelName = instanceof $model => User | Task ..
- $validatorName = $modelName . 'Validator';
- $validator = new $validatorName;
- return $validator->validate($model);